### PR TITLE
MAINTAINERS: posix: add cfriedt as maintainer

### PR DESCRIPTION
**Pull request by cfriedt**
Adding myself as maintainer of the POSIX API layer.

Signed-off-by: Christopher Friedt <cfriedt@fb.com>

### CODE REVIEW

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -1416,7 +1416,9 @@ nRF52 BSIM:
         - "platform: nRF52 BSIM"
 
 POSIX API layer:
-    status: odd fixes
+    status: maintained
+    maintainers:
+        - cfriedt
```
Comment (stephanosio) [inline MAINTAINERS.yml:1421]: should remove yourself from `collaborators:` now that you are in `maintainers:`.

**Review by nashif — DISMISSED** [review]

**Comment by stephanosio** [comment]
> why the TSC label?

@nashif because that is the documented process for adding a maintainer?

**Review by enjiamai — DISMISSED** [review]

File: MAINTAINERS.yml (hunk 1 above)
Comment (cfriedt) [reply MAINTAINERS.yml:1421]: Good catch ;-)

**Comment by cfriedt** [comment]
@nashif, @enjiamai - please re-ack.

**Comment by cfriedt** [comment]
> > why the TSC label?
> 
> @nashif because that is the documented process for adding a maintainer?

Probably short notice for the agenda for today. I originally had it linked to a different PR, which was a bit noisy.

**Review by stephanosio — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by enjiamai — APPROVED** [review]

**Pull request merged**